docs/release-notes.md

```diff
@@ -9,6 +9,12 @@ meta:
 ---
 # Release Notes
 
+## 7.3.0
+### Bug Fixes and Improvements
+* Duplicated PagerDuty incidents were being sent out
```

Comment:
I'd suggest phrasing the bugs as `we fixed X` rather than `X was happening`

```suggestion
* Fixed issue where duplicated PagerDuty incidents were being sent out
```

Reply:
Sounds good! I'll update as suggested going forward (including this one)